### PR TITLE
Use the correct rag-augmented image in the sample yamls

### PR DESCRIPTION
**Pull request by bparees**
also delete a bunch of resource content we no longer need so
we can maintain just one set of resource yamls

## Type of change

- [x] Refactor
- [ ] New feature
- [x] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by bparees** [comment]
@nemerna ptal

**Comment by syedriko** [comment]
@bparees https://github.com/openshift/lightspeed-service/pull/506 took effect, tests pass.

File: examples/openshift-lightspeed-tls.yaml [hunk 1]
```diff
@@ -38,7 +38,7 @@ spec:
           secret:
             secretName: lightspeed-certs
       containers:
-      - image: quay.io/openshiftdemos/lightspeed-w-rag:latest
+      - image: quay.io/openshift/lightspeed-service-api-rag
```
Comment (syedriko) [inline examples/openshift-lightspeed-tls.yaml:41]: :latest?

File: examples/openshift-lightspeed-tls.yaml [hunk 2]
```diff
@@ -139,6 +139,9 @@ data:
           - name: gpt-4-1106-preview
           - name: gpt-3.5-turbo
     ols_config:
+      reference_content:
+        product_docs_index_path: "./vector-db/ocp-product-docs"
+        product_docs_index_id: product
```
Comment (syedriko) [inline examples/openshift-lightspeed-tls.yaml:144]: embeddings_model_path: ./embeddings_model?

File: examples/openshift-lightspeed-tls.yaml (hunk 1 above)
Comment (bparees) [reply examples/openshift-lightspeed-tls.yaml:41]: good catch, yes (though i believe it also defaults to latest)

File: examples/openshift-lightspeed-tls.yaml (hunk 2 above)
Comment (bparees) [reply examples/openshift-lightspeed-tls.yaml:144]: good point, added

**Comment by bparees** [comment]
@nemerna @syedriko updated

i'm going to hold this until i can re-verify the template works on a cluster tomorrow, but please /lgtm it if you're ok with it and i'll release the hold once i test it.

/hold

**Comment by bparees** [comment]
/hold cancel

tested this on my own cluster

File: examples/openshift-lightspeed-tls.yaml (hunk 2 above)
Comment (bparees) [reply examples/openshift-lightspeed-tls.yaml:144]: sorry, i thought i pushed this change but apparently not.  adding it here: https://github.com/openshift/lightspeed-service/pull/514

**Pull request merged**